common/api/protos/HStream/Server/HStreamApi.proto

```diff
@@ -100,6 +100,32 @@ service HStreamApi {
   // FIXME: (store)node related apis
   rpc ListNodes(ListNodesRequest) returns (ListNodesResponse) {}
   rpc GetNode(GetNodeRequest) returns (Node) {}
+
+  rpc ReadShard(ReadShardRequest) returns (ReadShardResponse) {}
```

Comment:
You need to upgrade the `__VERSION__` number in Line7 (https://github.com/hstreamdb/hstream/pull/932/files#diff-4111556a0856f4a9898526fa2cd071f8ca257707a5d253eb08a00351b397af71L7) if you change this proto file.

Reply:
> You need to upgrade the `__VERSION__` number in Line7 (

Ok, but which version should I update to, v0.2.0 or v0.1.1?